src/main/java/org/folio/circulation/domain/policy/LoanPolicy.java

```diff
@@ -175,7 +179,12 @@ public static LoanPolicy unknown(String id) {
 
       errorWhenReachedRenewalLimit(loan, errors);
 
-      if(errors.isEmpty()) {
+      if (hasDeclaredLostItem(loan)) {
```

Comment:
I think this check should go right before line # 163 - `final Result<DateTime> proposedDueDateResult =`

Reply:
Why do you think so?

Reply:
This method is used to calculate new due date for an item:
```
final Result<DateTime> proposedDueDateResult =
        determineStrategy(null, true, isRenewalWithHoldRequest, systemDate)
            .calculateDueDate(loan);
```
I think, it make sense to skip this method if we know that renew for item have to be rejected.

Reply:
That makes sense. Changed.